Modelica/Electrical/Analog/Examples/OpAmps.mo

```diff
@@ -999,6 +999,173 @@ The output of the integrator part opamp (opAmp2) is a triangular signal of also
+    connect(sourceVoltage2.p, voltageSensorSource.n) annotation (Line(points={{-100,
+            -30},{-100,-20},{-80,-20},{-80,-10}}, color={0,0,255}));
+    connect(sourceVoltage1.p, voltageSensorSource.p) annotation (Line(points={{-120,
+            -30},{-120,20},{-80,20},{-80,10}}, color={0,0,255}));
+    connect(sourceVoltage1.p, resistanceLoad.p) annotation (Line(points={{-120,-30},
+            {-120,40},{-130,40},{-130,60},{-120,60}}, color={0,0,255}));
+    connect(sourceVoltage2.p, resistanceLoad.n) annotation (Line(points={{-100,-30},
+            {-100,40},{-90,40},{-90,60},{-100,60}}, color={0,0,255}));
+    connect(resistor3.n, resistanceInstrument.p) annotation (Line(points={{0,30},
+            {20,30},{20,0},{60,0},{60,-20}}, color={0,0,255}));
+    connect(resistor3.n, voltageSensor.p) annotation (Line(points={{0,30},{20,
+            30},{20,0},{80,0},{80,-20}}, color={0,0,255}));
+    connect(resistor3.n, opAmp.out)
+      annotation (Line(points={{0,30},{20,30},{20,0},{0,0}}, color={0,0,255}));
+    connect(resistor1.n, resistor3.p)
+      annotation (Line(points={{-40,30},{-20,30}}, color={0,0,255}));
+    connect(resistor2.n, resistor4.p)
+      annotation (Line(points={{-40,-30},{-20,-30}}, color={0,0,255}));
+    connect(resistor2.n, opAmp.in_p) annotation (Line(points={{-40,-30},{-30,
+            -30},{-30,-6},{-20,-6}}, color={0,0,255}));
+    connect(resistor1.n, opAmp.in_n) annotation (Line(points={{-40,30},{-30,30},
+            {-30,6},{-20,6}}, color={0,0,255}));
+    annotation (experiment(
+        StopTime=0.2,
+        Interval=0.0001,
+        Tolerance=1e-06),
+      Diagram(coordinateSystem(extent={{-140,-100},{140,100}})),
+      Icon(coordinateSystem(extent={{-100,-100},{100,100}})),
+      Documentation(info="<html>
+<p>
+This model demonstrates a differential amplifier to attenuate the input voltage to a level that can be handled by the measurement instrument.
+</p>
+<p>
+Two sinusoidal source voltages with a phase shift of 120&deg; (e.g. two phases of a three-phase system) feed a load resistor. 
+The voltage at the load resistor has to be measured but is too high for direct measurement.
+</p>
+<h4>Notes</h4>
+<ul>
+<li>The output of the amplifier is inverted with respect to the differential input, as well as attenuated by the factor 1/data.k.</li>
+<li>The influence of the measurement instrument's input resistance on the measurement result is small.</li>
+<li>The current consumption of the amplifier inputs at the source circuit is small (resistor1.i and resistor2.i).</li>
+<li>The common of the source and the common of the amplifier are connected by a ground resistor. 
+In case this resistance is low, a small current is flowing between the amplifier's common and the source's common (resistorGround.i). 
+In case this resistance is high, the amplifier's common is floating with respect to the source's common with a high voltage (resistorGround.v).</li>
```

Comment:
```suggestion
In case this resistance is high, the amplifier's common is floating with respect to the source's common with a high voltage (<code>resistorGround.v</code>).</li>
```